### PR TITLE
Import Trello: set creation date for older Trello elements

### PR DESCRIPTION
**Pull request by xavierpriour**
Older Trello elements do not report creation activity. As a result we cannot compute their actual creation date. Currently this crashes the import (issue #417).

This PR changes the import to use the current timestamp in that case - actually, we now log all import actions with the same "current" timestamp, this helps to retrieve what actions belong to what import. 

We also set a few more elements (if the importer properly mapped Trello members to Wekan users):

- comments are now assigned to their author (it used to be the importer)
- members that were admin of the original board are now admin of the imported board
- cards userId is now set to the original user that created them

### CODE REVIEW

File: models/import.js [hunk 1]
```diff
@@ -292,31 +342,35 @@ class TrelloCreator {
...
-        userId: Meteor.userId(),
+        // We attribute the import to current user,
+        // not the creator of the original object
+        userId: this._user(),
       });
     });
   }
 
+  getAdmin(trelloMemberType) {
+    if (trelloMemberType === 'admin') {
```
Comment (mquandalle) [inline models/import.js]:
```es6
return trelloMemberType === 'admin';
```

File: models/import.js [hunk 2]
```diff
@@ -25,6 +34,39 @@ class TrelloCreator {
...
+  /**
+   * If dateString is provided,
+   * return the Date it represents.
+   * If not, will return the date when it was first called.
+   * This is useful for us, as we want all import operations to
+   * have the exact same date for easier later retrieval.
+   *
+   * @param {String} dateString a properly formatted Date
+   */
+  _now(dateString) {
```
Comment (mquandalle) [inline models/import.js]: :+1:

**Comment by xavierpriour** [comment]
done @mquandalle

**Comment by mquandalle** [comment]
Thank you very much for your reactivity Xavier, I’ll be able demo this feature tomorrow :-)

**Pull request merged**